pysat/tests/test_constellation.py

```diff
@@ -136,8 +137,7 @@ def setup(self):
         """
         self.testC = pysat.Constellation(name='test_add_similar')
         self.refC = pysat.Constellation([pysat.Instrument('pysat', 'testing',
-                                                          tag='ascend',
-                                                          clean_level='clean')])
```

Comment:
This line removed since it was over 80 characters and no cleaning options are available for the tests